tox.ini

```diff
@@ -12,6 +12,7 @@ deps=
     poetry
 
 commands =
+    python -m pip install -U pip
```

Comment:
You should use `download=true` instead to update pip:

https://github.com/pytest-dev/pytest/blob/e8504e04f314528bc02eba13d76048295812fac2/tox.ini#L118-L119